test/components/test-harness-component.html

```diff
@@ -7,13 +7,48 @@
   <br-input br-model="$ctrl.sysSlug"
     br-options="{
       name: 'slug',
-      label: 'Short Name'
-    }">
+      label: 'Email Address',
+      theme: 'material',
+      type: 'email',
+      showValidation: true
+    }" required>
     <br-input-help>
-      Enter a unique short name for this identity.
+      Enter a unique email for this identity.
     </br-input-help>
+    <br-input-validation-errors>
+      <p ng-show="$ctrl.identityForm.slug.$error.required">
```

Comment:
This should be utilizing the newer ng-messages method as shown in angular-authn-password.

Reply:
Matt, I made these changes but now I am not seeing the error messages. I compared the view code between agreement and autn password and do not seem to be missing anything. I believe I have all the necessary

Reply:
See my comment on test-harness-component.html.

Reply:
Turns out I was not formatting the 'html' correctly